spec/models/auction_spec.rb

```diff
@@ -30,34 +30,4 @@
       end
     end
   end
-
-  describe "#winning_bid" do
```

Comment:
ah I wasn't sure you had! probably related to the "organization" comment. looks like these test s *are* here and I can see that much more clearly now. THANKS!